src/lib/build.js

```diff
@@ -35,6 +35,7 @@ const getBuildOptions = ({ cachedConfig, options: { context, cwd, debug, dry, js
   cwd,
   featureFlags: {
     functionsBundlingManifest: true,
+    edge_functions_produce_eszip: true,
```

Comment:
There's an additional place where CLI talks to Netlify Build where we also need to pass the feature flag: https://github.com/netlify/cli/blob/main/src/commands/deploy/deploy.js#L413.

Reply:
Ahhhh thanks! Will add this now.

Also TIL regarding `feat:` creating a release 👍🏻

Reply:
Sorted, here: https://github.com/netlify/cli/pull/4791/commits/a358964b135a645fa47f1a411fe57e1b27c978f2